<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientQt</class>
 <widget class="QMainWindow" name="ClientQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="envoi">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer message</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="contenuMess">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="celcius">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fara">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>°F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hyg">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Hygrométrie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numCapteur">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="reponseServ">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="messErreur">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connect</sender>
   <signal>clicked()</signal>
   <receiver>ClientQt</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnect</sender>
   <signal>clicked()</signal>
   <receiver>ClientQt</receiver>
   <slot>onDisconnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>envoi</sender>
   <signal>clicked()</signal>
   <receiver>ClientQt</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>celcius</sender>
   <signal>clicked()</signal>
   <receiver>ClientQt</receiver>
   <slot>onSendCelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fara</sender>
   <signal>clicked()</signal>
   <receiver>ClientQt</receiver>
   <slot>onSendFarClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hyg</sender>
   <signal>clicked()</signal>
   <receiver>ClientQt</receiver>
   <slot>onSendHygClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onDisconnectButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onSendCelClicked()</slot>
  <slot>onSendFarClicked()</slot>
  <slot>onSendHygClicked()</slot>
 </slots>
</ui>
